docs/examples/README.md

```diff
@@ -29,7 +29,7 @@
 # Triton Examples
```

Comment:
Please run some kind of spellchecker on all the .md files, there are a lot of typos that are difficult to catch manually

Reply:
I ran spellcheck in the latest commit but will run it again.